lib/rust/ensogl/core/src/data/color/space/def.rs

```diff
@@ -389,6 +389,61 @@ impl Rgb {
         Self::new(r.into() / 255.0, g.into() / 255.0, b.into() / 255.0)
     }
 
+    /// Return a color if the argument is a string matching a `#RGB` or `#RRGGBB` format, where
+    /// `R`, `G`, `B` represent lower- or upper-case hexadecimal digits. The `RR`, `GG`, `BB` color
+    /// components are mapped from `[00 - ff]` value range into `[0.0 - 1.0]` (a three-digit string
+    /// matching a `#RGB` format is equivalent to a six-digit string matching a `#RRGGBB` format,
```

Comment:
I would also allow format here NOT starting with `#` - could you please allow it here?